Codecademy/discourse-codebytes-plugin
# Proof of concept

Updates the Discourse renderer to show [codebyte] tags as the actual CodeByte editor inside an iframe.

See https://github.com/codecademy-engineering/static-sites/pull/608 for the corresponding changes in Static Sites to create the page that the iframe points at.

### Testing Instructions
1. Run static sites with `yarn develop` in that directory
2. Run Discourse with `bundle exec rails server` in this directory
3. Visit http://localhost:3000/
4. Click the "New Topic" button to start a new post
5. Click the "C" button to add a new CodeByte
6. The CodeByte editor appears in the preview!
![Screen Shot 2021-03-02 at 2 48 01 PM](https://user-images.githubusercontent.com/4821431/109705934-56519580-7b66-11eb-9f60-05b108dfbee4.png)
